tests/default_vars.sh

```diff
@@ -483,6 +484,7 @@ export DATM=false
 export DAYS="1"
 export FHMAX="24"
 export FDIAG="6"
+export FHZERO="6"
```

Comment:
ditto

Reply:
I looked at the tests, and saw none that would be affected. Also, the regression tests passed on Hera for rt.conf and rt_gnu.conf.